frontend/public/components/_dropdown.scss

```diff
@@ -66,7 +66,7 @@ $dropdown-background--hover: $pf-color-black-200; // pf-c-dropdown__menu-item--h
   white-space: nowrap;
 
   @media (min-width: $pf-global--breakpoint--xl) {
-    padding-left: $pf-global-gutter--md;
+    padding-left: $pf-global--spacer--sm;
```

Comment:
This change will affect all the dropdowns in the console. You need to change it for only `ApplicationDropdown`.

See it affect the projects dropdown. The project dropdown is now aligned to left.
![image](https://user-images.githubusercontent.com/2561818/159864291-ecdfcf98-718d-48c8-aef8-c872f20e7ff1.png)

Reply:
I have fixed it. Used the React Classnames Conditional for applying the CSS fix only to that page

Reply:
Hey @Lucifergene sorry but it looks still like on the screenshot by Vikram:

![image](https://user-images.githubusercontent.com/139310/163345239-46afc170-7872-4b15-b369-e43ae830830c.png)

![image](https://user-images.githubusercontent.com/139310/163345277-0dbd77e9-6ea6-4221-adb9-297ce756e408.png)

Reply:
Sorry for that! 
Have corrected the CSS